### PR TITLE
Scan: :zap: Implemented realtime price function

### LINKED ISSUE #2244: Support realtime update on `PriceHook.re`
Let's make `PriceHook.re` update realtime by using `AxiosHooks.useWithReload`.

### PR DESCRIPTION
**Pull request by pzshine**
Fixed: #2244

## Implementation details

 - Implemented the price realtime function

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [ ] The pull request includes a description of the implementation/work done in detail.
- [x] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: scan/src/contexts/GlobalContext.re [hunk 1]
```diff
@@ -4,13 +4,21 @@ let context = React.createContext(ContextHelper.default);
...
+  let (financialOpt, reload) = PriceHook.get();
 
   let data = {
     let%Opt financial = financialOpt;
     Some({financial: financial});
   };
 
+  React.useEffect1(
+    () => {
+      let intervalID = Js.Global.setInterval(() => {reload()}, 2000);
```
Comment (evilpeach) [inline scan/src/contexts/GlobalContext.re]: I think we should set an interval time to 1 minute.

Comment (evilpeach) [inline scan/src/contexts/GlobalContext.re]: Remove unnecessary `{`. Let's use `reload`

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: scan/src/contexts/GlobalContext.re (hunk 1 above)
Comment (sorawit) [reply scan/src/contexts/GlobalContext.re]: What's the type of `reload`? Can we just do ` Js.Global.setInterval(reload, ...`?

File: scan/src/rpc/PriceHook.re [hunk 2]
```diff
@@ -21,17 +21,27 @@ module Price = {
...
+  let (bandJson, bandReload) =
+    AxiosHooks.useWithReload("https://api.coingecko.com/api/v3/coins/band-protocol");
+  let reload = () => {
+    usdReload((), ());
+    btcReload((), ());
+    bandReload((), ());
+  };
+
+  let data =
+    switch (usdJson, btcJson, bandJson) {
```
Comment (sorawit) [inline scan/src/rpc/PriceHook.re]: This is fine. Or we can keep using `let%Opt`. up to you.
```
let data = {
   let%Opt ...
   ...
   Some(...)
}
```

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Review by evilpeach — APPROVED** [review]

**Review by sorawit — APPROVED** [review]

**Pull request merged**